packages/components/checkbox/src/checkbox.js

```diff
@@ -26,12 +29,22 @@ export class TSCheckbox extends TSElement {
 		return this.dir || this.bodyDir;
 	}
 
+	onClick(e) {
+		e.stopPropagation();
+		if (this.disabled) {
+			return;
+		}
+		this.checked = !this.checked;
+		this.dispatchCustomEvent(customEventNames.CHECKBOX_CHANGED, this.checked);
```

Comment:
I think it's a good idea to send the original event object beside the new checked value.

Reply:
Like this one : https://github.com/Tradeshift/elements/blob/1a3c45c188d00116446f0ce7bb2d14c8983d78a2/packages/components/file-uploader-input/src/file-uploader-input.js#L147